projects/flow.gmx-lysozyme-in-water/project.py

```diff
@@ -96,16 +94,10 @@ def _grompp_str(op_name, gro_name, checkpoint_file=None):
     return workspace_command(cmd)
 
 
-def _mdrun_str(op_name, np=1, nt=None, verbose=False):
+def _mdrun_str(op_name, nt=None, verbose=False):
     """Helper function, returns mdrun command string for operation."""
     num_threads = 1 if nt is None else nt
-    num_nodes = np // num_threads
-    cmd = (
-        "OMP_NUM_THREADS={num_threads} {mpi_exec} -n {np} {gmx} "
-        "mdrun -ntomp {num_threads} {verbose} -deffnm {op}"
-    ).format(
-        np=num_nodes,
-        mpi_exec=mpi_exec,
+    cmd = ("{gmx} mdrun -ntomp {num_threads} {verbose} -deffnm {op}").format(
```

Comment:
Why did this drop the `OMP_NUM_THREADS` environment variable?

Reply:
Because I meant to set the directive but forgot. `OMP_NUM_THREADS` is handled at the command generation level.

Reply:
Also because, `np` and `nt` are never used in the project.